requirements/requirements.in

```diff
@@ -1,59 +1,57 @@
-apache-libcloud==2.0.0
+apache-libcloud==2.2.1
 appdirs==1.4.2
-asgi-amqp==1.1.1
 asgiref==1.1.2
-azure==2.0.0rc6
+azure==3.0.0
 backports.ssl-match-hostname==3.5.0.1
-boto==2.46.1
-boto3==1.4.4
+boto==2.47.0
+boto3==1.6.2
 channels==1.1.8
 celery==3.1.25
-daphne==1.3.0
-Django==1.11.7
+daphne==1.3.0   # Last before backwards-incompatible channels 2 upgrade
+Django==1.11.10
 django-auth-ldap==1.2.8
 django-celery==3.2.2
-django-crum==0.7.1
-django-extensions==1.7.8
+django-crum==0.7.2
+django-extensions==2.0.0
 django-jsonfield==1.0.1
 django-oauth-toolkit==1.0.0
-django-polymorphic==1.3
+django-polymorphic==2.0.2
 django-pglocks==1.0.2
-django-radius==1.1.0
-django-solo==1.1.2
-django-split-settings==0.2.5
-django-taggit==0.22.1
-djangorestframework==3.7.3
+django-radius==1.3.2
+future==0.16.0      # for unlisted requirement django-radius/issues/15
+django-solo==1.1.3
+django-split-settings==0.3.0
+django-taggit==0.22.2
+djangorestframework==3.7.7
 djangorestframework-yaml==1.0.3
-gevent-websocket==0.9.5
-irc==15.1.1
+irc==16.2
 jsonschema==2.6.0
-M2Crypto==0.25.1
-Markdown==2.6.7
+M2Crypto==0.29.0
+Markdown==2.6.11
 ordereddict==1.1
 pexpect==4.4.0
 psphere==0.5.2
-psutil==5.2.2
-psycopg2==2.7.3.2
+psutil==5.4.3
+psycopg2==2.7.3.2   # problems with Segmentation faults / wheels on upgrade
 pycrypto==2.6.1
-pygerduty==0.35.2
-pyOpenSSL==17.0.0
+pygerduty==0.37.0
+pyOpenSSL==17.5.0
 pyparsing==2.2.0
 python-logstash==0.4.6
-python-memcached==1.58
+python-memcached==1.59
 python-radius==1.0
-python-saml==2.2.1
-social-auth-core==1.5.0
-social-auth-app-django==2.0.0
+python-saml==2.2.2      # pinned due to open issue awx/issues/1421
+social-auth-core==1.7.0
+social-auth-app-django==2.1.0
 pyvmomi==6.5
 redbaron==0.6.3
+requests<2.16  # Older versions rely on certify
 requests-futures==0.9.7
-service-identity==16.0.0
-shade==1.20.0
-slackclient==1.0.6
+service-identity==17.0.0
+slackclient==1.1.2
 tacacs_plus==1.0
-twilio==6.1.0
-twisted==17.9.0
```

Comment:
@wwitzel3 Halp! I don't know what I'm doing.

The websockets still work after I tested this. Does that mean we just don't need twisted and the gevent library anymore?

I'm pretty lost as to the necessity of these things on my own.

Reply:
I'd be surprised if we still needed gevent or twisted anymore.

Reply:
No, we do not need them